learning_resources/moderating-oss-repos.md

```diff
@@ -0,0 +1,342 @@
+
+Generally, a warning is something given in good faith. You believe someone has
+made an error, isn't self-aware, may be having a bad day or is otherwise going
+to be receptive to the warning. It's important to differentiate between
+behaviors that are disruptive (like re-opening an issue in disagreement), and
+those which have an impact (intended or not) on people.
+
+[Here is a sample warning notice](coc-communication-templates/warning.md).
+
+### Temporary Bans/Cooling Off Periods
+
+Like Warnings, there is **a huge leap of faith** on your part that the person
+given the decision will be receptive to a second-chance. It can be a good
+opportunity if someone is experiencing external stress in their lives to take
+time, reflect and return, but this option isn't recommended unless you have this
+level of insight. Statistically even with this thoughtful evaluation, temporary
+bans have about a 50/50 success rate.
+
+Here is an example of a
+[temporary ban notice](./coc-communication-templates/temporary-ban.md), which
+includes an on-boarding before return. .
+
+### Full Ban
+
+A full ban is just as it sounds - removal from all participation in all project
+spaces. This might occur after a warning was unsuccessful. Or, because an
+account was obviously created to spam - or someone is just so clearly showing no
+intent to participate professionally, and inclusively.
+
+This is the time to revisit all of the different spaces your project
+collaborates on and with.
+
+[Here is a sample ban notice](./coc-communication-templates/ban.md).
+
+### Immediate Ban
+
+If someone looks like a 'drive by' participant in your project or community, and
+by that we mean their interactions have only been disruptive, spammy, harmful or
+otherwise nuisance, reach out to [your escalation path]() to request their account be banned.
+
+## :hammer: Your Enforcement Plan
+
+Use this as a workspace to detail the steps your team will follow in event
+someone displays any of the discouraged behaviors in the Code of Conduct, or as
+listed in your CONTRIBUTING.md/README.md
+
+### Decision Making Process
+
+**Who should be consulted? Who will make the final decision if there is a
+disagreement?**
+
+It's good to have someone responsible and trusted to sign-off on decisions, when
+it makes sense. If you are unsure, or there is a tension or anything which feels
+out of scope for your team to manage, it probably is out of scope, and you
+should reach out to your manager, or skip level following the same process
+identified at the top of this document.
+
+### Examples
+
+Here are some examples of how you might, as a team, document your approach. It's
+important to remember, that no incident will be the same, and thus some of these
+will need to be tweaked for the situation. Nonetheless it's good to have a plan!
+
+**For violations where an employee (on our team or not) is reporting or being
+repeated.**
+
+- We will follow the guidance given, and not handle this situation.
```

Comment:
This could be read as "ignore the situation if its an employee" so maybe edit the line to be more explicit and say `We will follow the guidance given by our organization's HR department, and not attempt to handle the situation ourselves.`

Reply:
I just removed this part